cha-ching-app/src/Chevron.js

```diff
@@ -0,0 +1,103 @@
+const Chevron = () => {
```

Comment:
we may want to thicken the line of the circle on this SVG later as another design iteration